NEWS

```diff
@@ -9,6 +9,27 @@ Deprecated:
     * `mongoc_cursor_set_hint` is deprecated for `mongoc_cursor_set_server_id`
     * `mongoc_cursor_get_hint` is deprecated for `mongoc_cursor_get_server_id`
 
+
+
+libmongoc 1.27.3
+================
+
+Notes:
+
+  * This patch release introduces changes surrounding the development processes,
+    including new code signing and dependency tracking. There are no behavioral
+    or API changes.
+
+Fixes:
+
+  * Upgraded bundled Zlib from 1.12.3 to 1.13.1
```

Comment:
```suggestion
  * Upgraded bundled Zlib from 1.2.3 to 1.3.1
```

Reply:
Danget. That ended up in the release too :(